eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/src/ManagementClientGenerator.cs

```diff
@@ -51,6 +51,7 @@ protected override void Configure()
             AddMetadataReference(MetadataReference.CreateFromFile(typeof(ArmClient).Assembly.Location));
             AddVisitor(new RestClientVisitor());
             AddVisitor(new ResourceVisitor());
+            AddVisitor(new SystemObjectTypeVisitor());
```

Comment:
we should update this name accordingly as well

Reply:
renamed